mlc_llm/relax_model/llama.py

```diff
@@ -803,23 +803,35 @@ def get_model(args, hf_config):
             )
 
     mod, pidx2pname = param_manager.quantization_transform(mod)
-    pname2binname = load_torch_pname2binname_map(
-        model_path,
-        set(pidx2pname.values()),
-        excluded_params=["cos_cached", "sin_cached"],
-    )
-
     device = tvm.cpu()
-
     param_list = [None] * len(pidx2pname)
     assert len(param_list) == len(pidx2pname)
+    if args.quantization.pre_quantized:
+            param_list = args.quantization.load_quantized_params(
+                model_path,
+                param_list,
+                pidx2pname,
+                device,
+                excluded_params=["cos_cached", "sin_cached"],
+            )
```

Comment:
Looks like here we have some additional indents.

Reply:
resolved, thanks.